packages/launchpad/src/use-tracking/test/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { renderHook } from '@testing-library/react';
+import { useTracking } from '..';
+import { buildTask } from '../../test/lib/fixtures';
+
+jest.mock( '@automattic/calypso-analytics' );
+
+describe( 'useTracking', () => {
+	beforeEach( () => jest.clearAllMocks() );
+
+	const buildDefaultProps = ( options = {} ) => ( {
+		checklistSlug: 'site-setup-checklist',
+		context: 'customer-home',
+		siteIntent: 'build',
```

Comment:
I think we should include a test for the case where `siteIntent` is not defined, as that is a real-world case we should handle smoothly.

Reply:
Test added here https://github.com/Automattic/wp-calypso/pull/87141/commits/266290ecb659fc4decc755e510a829d381b1bee0